server/GraknServer.java

```diff
@@ -214,6 +216,10 @@ public static void main(String[] args) {
                 MigratorClient migrator = new MigratorClient(exportDataCommand.port());
                 boolean success = migrator.exportData(exportDataCommand.database(), exportDataCommand.filename());
                 System.exit(success ? 0 : 1);
+            } else if (command.isGetSchema()) {
```

Comment:
Now that we have three different commands here, I think it would make sense to extract each `if` block into its own method. Something like: `runGraknServer`, `importData`, `getSchema`.

Reply:
could do.